draft-ietf-quic-qpack.md

```diff
@@ -963,6 +963,9 @@ followed by the `S` bit indicating whether the reference is into the static
 field is represented as an integer with a 6-bit prefix (see Section 5.1 of
 [RFC7541]).
 
+If the entry in the static or dynamic table does not provide any header field
+value (as opposed to a name), a zero-length string MUST be used.
+
```

Comment:
Setting aside the legacy HPACK table that had nothing at index 0 (and my implementation did indeed return `(String.Empty,String.Empty)`), is this even possible?  If there's a reference to an index which isn't valid, that's an error.  If it is valid, then something has been inserted there -- which might have been a pair of empty strings, but that's still the contents.

Reply:
I think the "problem case" is referencing a static table item with empty value. We discussed how to build the header list and felt that it it the job of the H3 endpoint to validate the header list, not QPACKs. This change makes it clear what to do, and what H3 could check in validation.

Reply:
I think that it's easiest to say that entries in the static table all have a value, but that some values are empty.  This text seems to confer some sort of special status on "absent" values.

Reply:
@martinthomson I reworded that part and moved it to the "Static Table" section.